content/pages/0_index_fr.html

```diff
@@ -0,0 +1,87 @@
+<html lang="fr">
+
+<head>
+    <title>GeoMapFish</title>
+    <meta name="tags" content="geoportal, geomapfish, webgis, gis, python" />
+    <meta name="date" content="2016-10-14" />
+    <meta name="modified" content="2022-05-03" />
+    <meta name="authors" content="Michael Kalbermatten, Rémi Bovard" />
+    <meta name="URL" content="index.html" />
+    <meta name="Save_as" content="index.html" />
+    <meta name="Slug" content="0_index" />
+    <meta name="Lang" content="fr" />
+    <meta name="Menulabel" content="Home" />
+    <meta name="summary" content="GeoMapFish is a web based gis application" />
+</head>
+
+<body>
+    <div class="row">
+        <div class="col-sm-12">
+            <video autoplay muted class="img-responsive" loop
+                src="https://sitn.ne.ch/web/geomapfish/geomapfish-org/intro.mp4">
+            </video>
+        </div>
+    </div>
+    <div class="row" style="margin-top: 1em;">
+        <div class="col-sm-6">
+            <a href="https://geomapfish-demo-2-7.camptocamp.com" target="_blank" role="button" class="btn btn-primary btn-lg btn-block">Live demo</a>
+        </div>
+        <div class="col-sm-6">
+            <a href="https://github.com/geomapfish/getting_started" target="_blank" role="button" class="btn btn-default btn-lg btn-block">Getting started</a>
+        </div>
+    </div>
+    <p class="mt-2">
+        Bienvenue sur le site web de la communauté GeoMapFish (GMF), merci de votre visite !
+    </p>
+
+    <h2>Généralités</h2>
+
+    <p>
+        GeoMapFish permet de construire des applications WebSIG de manière aisée et flexible à travers un géoportail en
+        ligne. Cette solution a été développée pour répondre aux attentes de multiples acteurs, qu'ils soient de type
+        privés, publics ou académiques du domaine du géospatial.
+    </p>
+    <p>
+        GeoMapFish est organisé autour d'une <a href="./community">communauté et d'un project steering committee</a>.
+    </p>
+
+    <h2>Caractéristiques</h2>
+
+    <p>
+        Pour une liste plus complétes des caractéristiques, merci de vous référer à la page
+        des <a href="./functionalities">fonctionnalités</a>.
+    </p>
+    <div class="mt-1 flex">
+        <img class="mr-1 no-shrink" alt="webmapping" src="/images/3_2_webmapping.png">
```

Comment:
Is it possible to have the same width (40px) for all this images?

Reply:
Yes, but not part of this PR :) I'll open an issue for that

Reply:
Okay okay ^^